src/main/java/de/dennisguse/opentracks/util/CalendarUtil.java

```diff
@@ -0,0 +1,72 @@
+package de.dennisguse.opentracks.util;
+
+import android.content.Context;
+import android.text.format.DateUtils;
+import android.util.Log;
+
+import java.util.Calendar;
```

Comment:
Not a fan of Calendar.
Could we use the newer classes for date handling?

Reply:
BTW: What do you do to keep up with Java news? Any webpage/resource/RSS?

I didn't know there were better classes to use instead of Calendar.